test/web/controllers/questionnaire_page_test.exs

```diff
@@ -22,7 +22,7 @@ defmodule Advisor.Web.RequestPageTest do
     # maybe split this later into group_lead and advisor
     assert response
              |> Floki.find(".people-picker li")
-             |> length == 29
+             |> length == 27
```

Comment:
This test is kind of really fragile as it is really related to the local database content